### DISCUSSION TITLE
Group 8 discussion

### DISCUSSION THREAD

**Discussion by peter-fs** [Digital-Preservation-Coalition/PythonStudyGroups#17] [General]
Following Cynde's suggestion I'm starting our own Group 8 discussion.

Action items:
- try the batch rename project: https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/blob/main/Use%20Case%20-%20How%20to%20batch%20rename%20files.md
- identify one or more personal use cases – perhaps something relatively simple for now

**Comment by davecgibson** [comment]
Hi all! Just wanted to add the script for the CSV to XML process that I will be discussing this evening. Looking forward to chatting with you all!
[Dave-MICopyrightDescriptionProject-DMToolXML.txt](https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/files/14763722/Dave-MICopyrightDescriptionProject-DMToolXML.txt)

**Reply by davecgibson** [reply]
Hi all! Last night Peter and I went over an example of the CSV to XML scripts that I work with and I think (or hope) we both learned something in the process as I had also never really tried to puzzle out each line of the code before last night. Here are the csv and resulting XML that we used to test the script example that I posted yesterday. Lookling forward to seeing what people have been working on next month!
[python_study_group_test_1.txt](https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/files/14774535/python_study_group_test_1.txt)
[python_study_group_test_1.csv](https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/files/14774536/python_study_group_test_1.csv)

**Reply by peter-fs** [reply]
Hi Dave! Thanks again for your explanation of this script at the last session. The output file is very helpful and I hope to spend a little time on it myself and get a sense of how the script and output relate. Much appreciated!
